src/constants/translations/ua/guest-home-page.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
First of all, it's Ukrainian, and do we need it right now at all? I don't think so.

Reply:
Fixed.